index.js

```diff
@@ -0,0 +1,63 @@
+import Book from './modules/book.js';
+import startTime from './modules/datetime.js';
+
+const form = document.querySelector('#book-form');
+const list = document.querySelector('#list');
+
+// The is the event handler that submit the form
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+
+  if (title.trim() !== '' || author.trim() !== '') {
+    Book.addbook(title, author, Book.generateId());
+    const inputs = document.querySelectorAll('input');
+    inputs.forEach((inputs) => { inputs.value = ''; });
+  }
+});
+
+document.querySelector('#list').addEventListener('click', (e) => {
+  Book.deleteBookfromUI(e.target);
+  const targetID = e.target.getAttribute('data-id');
+  Book.removeBook(targetID);
+});
+
+// This is the part that loads the objects
+function init() {
+  if (Book.getBooks().length === 0) {
+    list.innerHTML = '';
+  } else {
+    Book.getBooks().forEach((book) => Book.displaybooks(book, list));
+  }
+}
+
+init();
+startTime();
+
+// -----Navbar functionality-----//
+
+const aBookList = document.querySelector('#a-book-list');
+const aAddBook = document.querySelector('#a-add-book');
+const aContact = document.querySelector('#a-contact');
```

Comment:
- [ ] Please I can't access the contact page, make sure the link to the `Contact page` on the Navigation bar is working.

Reply:
Cantact page is also working